build.assets/Dockerfile-build

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:22.04
```

Comment:
Can you add some docs/comments on what this dockerfile and the other do? We have so many under `build.assets/**` that I don't even know what they all do anymore.